lib/kb_muscle/kb_muscleImpl.py

```diff
@@ -455,7 +455,10 @@ def MUSCLE_nuc(self, ctx, params):
                 if genome_type == 'KBaseGenomes.Genome':
                     genome = genome_obj['data']
                     genomeSciName[genomeRef] = genome['scientific_name']
-                    for feature in genome['features']:
+
+                    all_features = genome['features'] + genome['non_coding_features']
```

Comment:
Will `non_coding_features` always be present in the genome object?

Reply:
changed to genome.get()

Reply:
oh. I think you should default the return of get to an empty list. It will still error in case one of them is none, e.g.  `None + a list`.

Reply:
added default empty list